docs/containers/quickstart-node.md

```diff
@@ -0,0 +1,118 @@
+
+1. Open the project folder in VS Code
+1. Open the Command Palette (`kb(workbench.action.showCommands)`) and use `Docker: Add Docker Files to Workspace...` command:
+
+   ![Add Dockerfile to a Node.js project](images/quickstarts/node-add-node.png)
+
+1. Select `Node.js` when prompted for the application platform
+1. Select either `Yes` or `No` when prompted to include Docker Compose files
+
+   > The Docker Compose files are optional and not used for debugging the application within a container, so either is a valid choice.
+
+1. Enter `3000` when prompted for the application port
+
+The extension will create `Dockerfile` and `.dockerignore` files. If you elected to include Docker Compose files, `docker-compose.yml` and `docker-compose.debug.yml` will be generated as well. Finally, the extension will create a set of **VS Code tasks** in `.vscode/tasks.json` for building and running the container (in both debug- and release-configurations) and a **launch debug configuration** in `.vscode/launch.json` for debugging the service within the container.
+
+## Run the service locally
+
+1. Open a terminal (`kb(workbench.action.terminal.toggleTerminal)`)
+1. Enter `npm run start` to start the application:
+
+   ```output
+   > express-app@0.0.0 start /Users/user/code/scratch/express-app
+   > node ./bin/www
+   ```
+
+1. Open the web browser and navigate to [http://localhost:3000](http://localhost:3000). You should see a page similar to the following:
+
+   ![Application page in browser](images/quickstarts/node-run-browser.png)
+
+1. When done testing, type `Ctrl+C` in the terminal
+
+## Build the service image
+
+1. Open the Command Palette (`kb(workbench.action.showCommands)`) and select the `Docker Images: Build Image...` command
+1. Open the Docker view and verify that the new image is visible in the Images tree:
+
+   ![Verify Docker image exists](images/quickstarts/node-verify-image.png)
+
+## Run the service container
+
+1. Right-click on the image built in the previous section and select `Run` or `Run Interactive`. The container should start and you should be able to see it in the Docker Containers tree:
+
+   ![Running service container](images/quickstarts/node-running-container.png)
+
+1. Open the web browser and navigate to [http://localhost:3000](http://localhost:3000). You should see a page similar to the following:
+
+   ![Application page in browser](images/quickstarts/node-run-browser.png)
+
+1. When done testing, right-click the container in the Containers tree and select `Stop`
+
+## Debug in the service container
+
+When Docker files were added to the application, the Docker extension also added a **VS Code debugger configuration** in `.vscode/launch.json` for debugging the service when running inside a container. The extension will detect the protocol and port used by the service and point the browser to the service, but the application needs to ensure that its output is written to the debug console:
```

Comment:
How about changing the tense here:
When the Docker extension adds Docker files to the application, it also adds a …
The extension detects the protocol and port used …

Also I'd suggest:
but make sure that your application's output is written to the debug console.

Reply:
Sounds good to me!  Updated.